Formula/mysql.rb

```diff
@@ -0,0 +1,186 @@
+class Mysql < Formula
+  desc "Open source relational database management system"
+  homepage "https://dev.mysql.com/doc/refman/8.0/en/"
+  url "https://cdn.mysql.com/Downloads/MySQL-8.0/mysql-boost-8.0.32.tar.gz"
+  sha256 "1a83a2e1712a2d20b80369c45cecbfcc7be9178d4fc0e81ffba5c273ce947389"
+  license "GPL-2.0-only" => { with: "Universal-FOSS-exception-1.0" }
+
+  livecheck do
+    url "https://dev.mysql.com/downloads/mysql/?tpl=files&os=src"
+    regex(/href=.*?mysql[._-](?:boost[._-])?v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  bottle do
```

Comment:
Bottles might not be hosted indefinitely so we probably shouldn't rely on those upstream. Not urgent though.

Reply:
Good point. We may or may not choose to run with this formula long term. I'll upload our two key bottles now.